Playbooks/Prompt-User.json

```diff
@@ -0,0 +1,331 @@
+
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata":{
+        "comments": "This playbook will ask the user if they completed the action from the Incident in Azure Sentinel.  If so, it will close the incicdent and add a comment.  If not, it will post a message to teams for the SOC to investigate and add a comment to the incident.",
```

Comment:
typo - "incicdent", should be "incident".  Also, I think "teams" should be capitalized and you may want to say "Microsoft Teams".